docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
     - lnd2
     - lnd-outside-1
     - lnd-outside-2
+    - otel-agent
```

Comment:
I don't think `otel-agent` is a dependency of integration tests is it?

Reply:
it was added on purpose to get telemetry when doing `make integration`

Reply:
won't do anything unless the env var are being set. but maybe that would be useful to get this in the CI server to see the difference between the tests over time. like if we introduce regression we would could it?

Reply:
like this trace is interesting/intriguing: https://ui.honeycomb.io/galoy/datasets/nicolas-dev/result/oi9y9hjhAqF?tab=raw the consistent 3s (timeout?) could highlight a bug